src/Silk.Core.Discord/Commands/Experimental/CastingDictionaryTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
```

Comment:
Interesting, I guess those will be removed in prod?

Reply:
Definitely. Forgot to remove this. Just wanted to see how much memory it would take up if something bad were to happen, and the properties (which re-cast) were to be accessed recursively.

Reply:
Fixed.